spec/text_message_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'text_message'
```

Comment:
It would be worth having another go at testing this at some point.  The practice will come in handy.